
<!DOCTYPE html>
<html>
  <head><title>CSV2HTML: tcrd210225_data_dictionary-Current_Variables.tsv</title></head>
  <style type="text/css">/* includes alternating gray and white with on-hover color */

.mystyle {
  font-size: 11pt; 
  font-family: Arial;
  border-collapse: collapse; 
  border: 1px solid silver;
}

.mystyle table {
  width: 100%;
}

.mystyle td, th {
  padding: 5px;
  max-width:400px;
  word-wrap:break-word;
  overflow-wrap: break-word;
}

.mystyle tr:nth-child(even) {
  background: #E0E0E0;
}

.mystyle tr:hover {
  background: silver;
  cursor: pointer;
}
</style>
  <body>
    <table border="1" class="dataframe mystyle">
  <thead>
    <tr style="text-align: center;">
      <th>Primary Table</th>
      <th>Column name</th>
      <th>Column description</th>
      <th>potential values; example value</th>
      <th>DATA TYPE</th>
      <th>distinct_values</th>
      <th>DATA SOURCE</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>alias</td>
      <td>id</td>
      <td></td>
      <td>1-150037</td>
      <td>int(11)</td>
      <td>150037</td>
      <td></td>
    </tr>
    <tr>
      <td>alias</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>1-19673</td>
      <td>int(11)</td>
      <td>19673</td>
      <td></td>
    </tr>
    <tr>
      <td>alias</td>
      <td>type</td>
      <td></td>
      <td>uniport, symbol</td>
      <td>enum('symbol','uniprot')</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>alias</td>
      <td>value</td>
      <td></td>
      <td>Q53FB3</td>
      <td>varchar(255)</td>
      <td>125167</td>
      <td></td>
    </tr>
    <tr>
      <td>alias</td>
      <td>dataset_id</td>
      <td>foreign key to column `id` on table dataset.</td>
      <td>1</td>
      <td>int(11)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>id</td>
      <td></td>
      <td>1-511408</td>
      <td>int(11)</td>
      <td>511408</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>6107</td>
      <td>int(11)</td>
      <td>2947</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>clinvar_phenotype_id</td>
      <td>foreign key to column id on table clinvar_phenotype.</td>
      <td>4077</td>
      <td>int(11)</td>
      <td>5803</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>alleleid</td>
      <td></td>
      <td>15046</td>
      <td>int(11)</td>
      <td>97381</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>type</td>
      <td></td>
      <td>unique types, single nucleotide variant, short repeat, deletion, duplication, indel, insertion, inversion, NT expansion, undetermined variant</td>
      <td>varchar(40)</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>name</td>
      <td></td>
      <td>NM_000410.3(HFE):c.187C&gt;G (p.His63Asp)</td>
      <td>text</td>
      <td>97381</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>review_status</td>
      <td></td>
      <td>criteria provided, multiple submitters, no conflicts, reviewed by expert panel</td>
      <td>varchar(60)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>clinical_significance</td>
      <td></td>
      <td>Pathogenic, other; Pathogenic; Pathogenic/Likely pathogenic; Benign; Likely pathogenic; drug response; Benign/Likely benign; Likely benign, other; Likely benign; Pathogenic/Likely pathogenic, other; Benign/Likely benign, other; Benign, other; Pathogenic/Likely pathogenic, risk factor; Uncertain significance, other; Likely benign, risk factor; Pathogenic/Likely pathogenic, drug response; Pathogenic, drug response; Benign/Likely benign, risk factor; Benign, drug response; Pathogenic, risk factor; Benign, risk factor; other, risk factor; Uncertain significance, association; drug response, other; Likely benign, drug response; Likely pathogenic, risk factor</td>
      <td>varchar(80)</td>
      <td>27</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>clin_sig_simple</td>
      <td></td>
      <td>1</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>last_evaluated</td>
      <td></td>
      <td>2019-01-09</td>
      <td>date</td>
      <td>1883</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>dbsnp_rs</td>
      <td></td>
      <td>1799945</td>
      <td>int(11)</td>
      <td>91635</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>dbvarid</td>
      <td></td>
      <td>nsv1067859, nsv1197557, nsv513781, nsv1197451, nsv513792, nsv1067916</td>
      <td>varchar(10)</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>origin</td>
      <td></td>
      <td>germline;inherited;unknown</td>
      <td>varchar(60)</td>
      <td>67</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>origin_simple</td>
      <td></td>
      <td>germline, germline/somatic, somatic, unknown</td>
      <td>varchar(20)</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>assembly</td>
      <td></td>
      <td>GRCh37, GRCh38, NCBI36</td>
      <td>varchar(8)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>chr</td>
      <td></td>
      <td>1-26, MT, X, Y, na</td>
      <td>varchar(2)</td>
      <td>26</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>chr_acc</td>
      <td></td>
      <td>NC_000006.12</td>
      <td>varchar(20)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>start</td>
      <td></td>
      <td>26090951</td>
      <td>int(11)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>stop</td>
      <td></td>
      <td>26090951</td>
      <td>int(11)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>integer_submitters</td>
      <td></td>
      <td>13</td>
      <td>int(2)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>tested_in_gtr</td>
      <td></td>
      <td>TRUE</td>
      <td>tinyint(1)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar</td>
      <td>submitter_categories</td>
      <td></td>
      <td>TRUE</td>
      <td>tinyint(1)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar_phenotype</td>
      <td>id</td>
      <td></td>
      <td>1-12280</td>
      <td>int(11)</td>
      <td>12280</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar_phenotype</td>
      <td>name</td>
      <td></td>
      <td>Nephronophthisis</td>
      <td>varchar(255)</td>
      <td>12279</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar_phenotype_xref</td>
      <td>id</td>
      <td></td>
      <td>1-22373</td>
      <td></td>
      <td>22373</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar_phenotype_xref</td>
      <td>clinvar_phenotype_id</td>
      <td>foreign key to column id on table clinvar_phenotype.</td>
      <td>1-10297</td>
      <td></td>
      <td>10297</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar_phenotype_xref</td>
      <td>source</td>
      <td></td>
      <td>Orphanet, SNOMED CT, MedGen, HPO, OMIM, MeSH, EFO, Gene</td>
      <td>tinyint(1)</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td>clinvar_phenotype_xref</td>
      <td>value</td>
      <td></td>
      <td>ORPHA79172</td>
      <td>tinyint(1)</td>
      <td>22361</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>id</td>
      <td></td>
      <td>4</td>
      <td>int(11)</td>
      <td>500993</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>3000</td>
      <td>int(11)</td>
      <td>2051</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>catype</td>
      <td>foreign key to column name on table cmpd_activity_type.</td>
      <td>ChEMBL, Guide to Pharmacology</td>
      <td>varchar(255)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>cmpd_id_in_src</td>
      <td>Source compound ID</td>
      <td>CHEMBL186240</td>
      <td>varchar(255)</td>
      <td>271016</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>cmpd_name_in_src</td>
      <td>Source compound name</td>
      <td>N-(5-Cyclobutyl-thiazol-2-yl)-2-(1H-indol-3-yl)-acetamide</td>
      <td>text</td>
      <td>283853</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>smiles</td>
      <td>SMILES</td>
      <td>O=C(Cc1c[nH]c2ccccc12)Nc3ncc(s3)C4CCC4</td>
      <td>text</td>
      <td>269547</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>act_value</td>
      <td>Bioactivity measurement value</td>
      <td>7.8</td>
      <td>decimal(10,8)</td>
      <td>1322</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>act_type</td>
      <td>Bioactivity measurement type</td>
      <td>IC50</td>
      <td>varchar(255)</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>reference</td>
      <td>Literature reference</td>
      <td>Bioorg. Med. Chem. Lett., (2004) 14:22:5521</td>
      <td>text</td>
      <td>16125</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>pubmed_ids</td>
      <td>PubMed IDs</td>
      <td>15482916</td>
      <td>text</td>
      <td>18504</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>cmpd_pubchem_cid</td>
      <td>PubChem CID</td>
      <td>9815202</td>
      <td>int(11)</td>
      <td>249460</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity</td>
      <td>lychi_h4</td>
      <td></td>
      <td>2DB427URCFNS</td>
      <td>varchar(15)</td>
      <td>266188</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity_type</td>
      <td>name</td>
      <td></td>
      <td>ChEMBL, Guide to Pharmacology</td>
      <td>varchar(255)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>cmpd_activity_type</td>
      <td>description</td>
      <td></td>
      <td>A manually curated chemical database of bioactive molecules with drug-like properties. It is maintained by the European Bioinformatics Institute (EBI), of the European Molecular Biology Laboratory (EMBL), based at the Wellcome Trust Genome Campus, Hinxton, UK. The IUPHAR/BPS Guide to PHARMACOLOGY</td>
      <td>text</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>id</td>
      <td></td>
      <td>2</td>
      <td>int(11)</td>
      <td>1075822</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>ctype</td>
      <td>foreign key to column name on table compartment_type.</td>
      <td>Human Cell Atlas, JensenLab Experiment, JensenLab Knowledge, JensenLab Prediction, JensenLab Text Mining</td>
      <td>varchar(255)</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>(null)</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>15604</td>
      <td>int(11)</td>
      <td>18813</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>go_id</td>
      <td></td>
      <td>GO:0005739</td>
      <td>varchar(255)</td>
      <td>2984</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>go_term</td>
      <td></td>
      <td>Mitochondria</td>
      <td>text</td>
      <td>3000</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>evidence</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>118</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>zscore</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(4,3)</td>
      <td>4715</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>conf</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(2,1)</td>
      <td>33</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>url</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>184602</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment</td>
      <td>reliability</td>
      <td></td>
      <td>Supported, Approved, Validated, (null)</td>
      <td>enum('Supported','Approved','Validated')</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment_type</td>
      <td>name</td>
      <td></td>
      <td>Human Cell Atlas, JensenLab Experiment, JensenLab Knowledge, JensenLab Prediction, JensenLab Text Mining</td>
      <td>varchar(255)</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>compartment_type</td>
      <td>description</td>
      <td></td>
      <td>Human Cell Atlas protein locations, minus rows with Uncertain reliability. Experiment channel subcellular locations from JensenLab COMPARTMENTS resource, filtered for confidence scores of 3 or greater. Knowledge channel subcellular locations from JensenLab COMPARTMENTS resource, filtered for confidence scores of 3 or greater. Prediction channel subcellular locations from JensenLab COMPARTMENTS resource, filtered for confidence scores of 3 or greater. Text Mining channel subcellular locations from JensenLab COMPARTMENTS resource, filtered for zscore of 3.0 or greater.</td>
      <td>text</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>id</td>
      <td></td>
      <td>1-407551</td>
      <td>int(11)</td>
      <td>407551</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>dtype</td>
      <td>foreign key to column name on table disease_type.</td>
      <td>CTD, DisGeNET, DrugCentral Indication, eRAM, Expression Atlas, JensenLab Experiment COSMIC, JensenLab Experiment DistiLD, JensenLab Knowledge GHR, JensenLab Knowledge UniProtKB-KW, JensenLab Text Mining, Monarch, UniProt Disease</td>
      <td>varchar(255)</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>1</td>
      <td>int(11)</td>
      <td>18879</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>nhprotein_id</td>
      <td>foreign key to column id on table nhprotein.</td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>name</td>
      <td></td>
      <td>Cystathioninuria</td>
      <td>text</td>
      <td>20032</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>ncats_name</td>
      <td></td>
      <td>Cystathioninuria</td>
      <td>text</td>
      <td>19683</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>did</td>
      <td></td>
      <td>MIM:219500</td>
      <td>varchar(20)</td>
      <td>28232</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>evidence</td>
      <td></td>
      <td>5 8</td>
      <td>text</td>
      <td>3225</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>zscore</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(4,3)</td>
      <td>4511</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>conf</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(2,1)</td>
      <td>37</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>description</td>
      <td></td>
      <td>Autosomal recessive phenotype characterized by abnormal accumulation of plasma cystathionine, leading to increased urinary excretion.</td>
      <td>text</td>
      <td>3887</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>reference</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>drug_name</td>
      <td></td>
      <td>cepharanthine</td>
      <td>text</td>
      <td>1443</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>log2foldchange</td>
      <td></td>
      <td>-2.329</td>
      <td>decimal(5,3)</td>
      <td>5245</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>pvalue</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>137079</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>score</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(16,15)</td>
      <td>83</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>source</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>135</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>O2S</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(16,13)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>disease</td>
      <td>S2O</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(16,13)</td>
      <td>5626</td>
      <td></td>
    </tr>
    <tr>
      <td>disease_type</td>
      <td>name</td>
      <td></td>
      <td>CTD, DisGeNET, DrugCentral Indication, eRAM, Expression Atlas, JensenLab Experiment COSMIC, JensenLab Experiment DistiLD, JensenLab Knowledge GHR, JensenLab Knowledge UniProtKB-KW, JensenLab Text Mining, Monarch, UniProt Disease</td>
      <td>varchar(255)</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td>disease_type</td>
      <td>description</td>
      <td></td>
      <td>descriptions</td>
      <td>text</td>
      <td>12</td>
      <td>Gene-Disease associations with direct evidence from the Comparative Toxicogenomics Database. Currated disease associations from DisGeNET (http://www.disgenet.org/). Disease indications and associated drug names from Drug Central. Currated gene disease associations from eRAM. Target-Disease associations from Expression Atlas where the log2 fold change in gene expresion in disease sample vs reference sample is greater than 1.0. Only reference samples "normal" or "healthy" are selected. Data is derived from the file: ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/atlas-latest-data.tar.gz, JensenLab Experiment channel using COSMIC, JensenLab Experiment channel using DistiLD, JensenLab Knowledge channel using GHR, JensenLab Knowledge channel using UniProtKB-KW, JensenLab Text Mining channel, Gene-Disease associations from Monarch that have S2O and/or O2S score(s), Disease association from UniProt comment field with type="disease"</td>
    </tr>
    <tr>
      <td>do</td>
      <td>doid</td>
      <td><a href="https://disease-ontology.org" target="_blank">https://disease-ontology.org</a></td>
      <td>DOID:0001816</td>
      <td>varchar(12)</td>
      <td>9233</td>
      <td><a href="https://disease-ontology.org" target="_blank">https://disease-ontology.org</a></td>
    </tr>
    <tr>
      <td>do</td>
      <td>name</td>
      <td></td>
      <td>angiosarcoma</td>
      <td>text</td>
      <td>9233</td>
      <td></td>
    </tr>
    <tr>
      <td>do</td>
      <td>def</td>
      <td></td>
      <td>A vascular cancer that derives_from the cells that line the walls of blood vessels or lymphatic vessels.</td>
      <td>text</td>
      <td>6118</td>
      <td></td>
    </tr>
    <tr>
      <td>do_parent</td>
      <td>doid</td>
      <td>foreign key to column doid on table do.</td>
      <td>DOID:0080337</td>
      <td>varchar(12)</td>
      <td>9232</td>
      <td></td>
    </tr>
    <tr>
      <td>do_parent</td>
      <td>parent_id</td>
      <td></td>
      <td>DOID:0050737</td>
      <td>varchar(12)</td>
      <td>2097</td>
      <td></td>
    </tr>
    <tr>
      <td>do_xref</td>
      <td>doid</td>
      <td>foreign key to column doid on table do.</td>
      <td>DOID:10588</td>
      <td>varchar(12)</td>
      <td>8635</td>
      <td></td>
    </tr>
    <tr>
      <td>do_xref</td>
      <td>db</td>
      <td></td>
      <td>CSP2005, DERMO, DO, EFO, GARD, ICD-10, ICD10CM, ICD9CM, ICD9CM_2006, JA, KEGG, ls, MEDDRA, MESH, MTH, MTHICD9_2006, NCI, NCI2004_11_17, OMIM, ORDO, sn, SNOMECT, SNOMED_CT_US_2018_03_01, SNOMEDCT_US_2018_03_01, stedman, UMLS_CUI</td>
      <td>varchar(24)</td>
      <td>25</td>
      <td></td>
    </tr>
    <tr>
      <td>do_xref</td>
      <td>value</td>
      <td></td>
      <td>2042-2215</td>
      <td>varchar(24)</td>
      <td>36392</td>
      <td></td>
    </tr>
    <tr>
      <td>drgc_resource</td>
      <td>id</td>
      <td></td>
      <td>1-320</td>
      <td>int(11)</td>
      <td>320</td>
      <td></td>
    </tr>
    <tr>
      <td>drgc_resource</td>
      <td>rssid</td>
      <td></td>
      <td><a href="https://repo.metadatacenter.org/template-instances/6b9c4cec-0c93-4b49-927b-6f81f55df901" target="_blank">https://repo.metadatacenter.org/template-instances/6b9c4cec-0c93-4b49-927b-6f81f55df901</a></td>
      <td>text</td>
      <td>320</td>
      <td></td>
    </tr>
    <tr>
      <td>drgc_resource</td>
      <td>resource_type</td>
      <td></td>
      <td>Cell, Mouse, Expression, Mouse Phenotype, NanoBRET, GPCR Mouse Imaging, Chemical Tool</td>
      <td>varchar(255)</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td>drgc_resource</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>1-149</td>
      <td>int(11)</td>
      <td>149</td>
      <td></td>
    </tr>
    <tr>
      <td>drgc_resource</td>
      <td>json</td>
      <td></td>
      <td>json file</td>
      <td>text</td>
      <td>320</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>id</td>
      <td>TCRD drug activity ID</td>
      <td>1-4004</td>
      <td>int(11)</td>
      <td>4004</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>1-993</td>
      <td>int(11)</td>
      <td>993</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>drug</td>
      <td>Drug name</td>
      <td>avapritinib</td>
      <td>varchar(255)</td>
      <td>1642</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>act_value</td>
      <td>Bioactivity measurement value</td>
      <td>9.62</td>
      <td>decimal(10,8)</td>
      <td>1019</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>act_type</td>
      <td>Bioactivity measurement type</td>
      <td>(null), IC50, Ki, Kd, EC50, ID50, ED50, MEC, IC90, A2, Kb, AC50</td>
      <td>varchar(255)</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>action_type</td>
      <td>Pharmacological action type</td>
      <td>INHIBITOR, PHARMACOLOGICAL CHAPERONE, AGONIST, ANTIBODY BINDING, ANTAGONIST, BLOCKER, ACTIVATOR, INVERSE AGONIST, MODULATOR, PARTIAL AGONIST, OPENER, POSITIVE ALLOSTERIC MODULATOR, NEGATIVE ALLOSTERIC MODULATOR, GATING INHIBITOR,, RELEASING AGENT, POSITIVE MODULATOR, SUBSTRATE, ALLOSTERIC ANTAGONIST, (null), BINDING AGENT, ALLOSTERIC MODULATOR, ANTISENSE INHIBITOR</td>
      <td>varchar(255)</td>
      <td>22</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>has_moa</td>
      <td>Mechanism of Action target association as defined by DrugCentral</td>
      <td>True, False</td>
      <td>tinyint(1)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>source</td>
      <td>Source of knowledge</td>
      <td>SCIENTIFIC LITERATURE, DRUG LABEL, KEGG DRUG, IUPHAR, CHEMBL</td>
      <td>varchar(255)</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>reference</td>
      <td>Source URL</td>
      <td><a href="https://www.accessdata.fda.gov/drugsatfda_docs/label/2020/212608s000lbl.pdf" target="_blank">https://www.accessdata.fda.gov/drugsatfda_docs/label/2020/212608s000lbl.pdf</a></td>
      <td>text</td>
      <td>1716</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>smiles</td>
      <td>SMILES</td>
      <td>CN1C=C(C=N1)C1=CN2N=CN=C(N3CCN(CC3)C3=NC=C(C=N3)[C@@](C)(N)C3=CC=C(F)C=C3)C2=C1</td>
      <td>text</td>
      <td>1495</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>cmpd_chemblid</td>
      <td>ChEMBL molecule ID</td>
      <td>CHEMBL4204794</td>
      <td>varchar(255)</td>
      <td>1513</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>nlm_drug_info</td>
      <td></td>
      <td>Avapritinib is a tyrosine kinase inhibitor that targets PDGFRA and PDGFRA D842 mutants as well as multiple KIT exon 11, 11/17 and 17 mutants with half maximal inhibitory concentrations (IC50s) less than 25 nM. Certain mutations in PDGFRA and KIT can result in the autophosphorylation and constitutive activation of these receptors which can contribute to tumor cell proliferation. Other potential targets for avapritinib include wild type KIT, PDGFRB, and CSFR1.</td>
      <td>text</td>
      <td>1473</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>cmpd_pubchem_cid</td>
      <td>PubChem CID</td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>(null)</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>dcid</td>
      <td>DrugCentral ID (drug compound)</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1642</td>
      <td></td>
    </tr>
    <tr>
      <td>drug_activity</td>
      <td>lychi_h4</td>
      <td></td>
      <td>BUBUSK1338HA</td>
      <td>varchar(15)</td>
      <td>1489</td>
      <td></td>
    </tr>
    <tr>
      <td>dto</td>
      <td>dtoid</td>
      <td></td>
      <td>BTO_0000007</td>
      <td>varchar(255)</td>
      <td>17779</td>
      <td></td>
    </tr>
    <tr>
      <td>dto</td>
      <td>name</td>
      <td></td>
      <td>HEK-293 cell</td>
      <td>text</td>
      <td>17733</td>
      <td></td>
    </tr>
    <tr>
      <td>dto</td>
      <td>parent_id</td>
      <td></td>
      <td>BTO_0000067</td>
      <td>varchar(255)</td>
      <td>2841</td>
      <td></td>
    </tr>
    <tr>
      <td>dto</td>
      <td>def</td>
      <td></td>
      <td>Established from a human primary embryonal kidney transformed by adenovirus type 5.'</td>
      <td>text</td>
      <td>5706</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>id</td>
      <td></td>
      <td>1-26935759</td>
      <td>int(11)</td>
      <td>26935759</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>etype</td>
      <td>foreign key to column name on table expression_type.</td>
      <td>CCLE, Cell Surface Protein Atlas, Consensus, GTEx, HCA RNA, HPA, HPM Gene, HPM Protein, JensenLab Experiment Cardiac proteome, JensenLab Experiment Exon array, JensenLab Experiment GNF, JensenLab Experiment HPA, JensenLab Experiment HPA-RNA, JensenLab Experiment HPM, JensenLab Experiment RNA-seq, JensenLab Experiment UniGene, JensenLab Knowledge UniProtKB-RC, JensenLab Text Mining, UniProt Tissue</td>
      <td>varchar(255)</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>3</td>
      <td>int(11)</td>
      <td>19945</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>tissue</td>
      <td></td>
      <td>Brain</td>
      <td>text</td>
      <td>5721</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>qual_value</td>
      <td></td>
      <td>(null), Not detected, Medium, High, Low</td>
      <td>enum('Not detected','Low','Medium','High')</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>integer_value</td>
      <td></td>
      <td>float</td>
      <td>decimal(12,6)</td>
      <td>4523984</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>boolean_value</td>
      <td></td>
      <td>TRUE, (null)</td>
      <td>tinyint(1)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>string_value</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>53449</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>pubmed_id</td>
      <td></td>
      <td>17974005</td>
      <td>int(11)</td>
      <td>10490</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>evidence</td>
      <td></td>
      <td>(null), CURATED, Approved, Enhanced, Supported</td>
      <td>varchar(255)</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>zscore</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(4,3)</td>
      <td>4046</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>conf</td>
      <td></td>
      <td>(null)</td>
      <td>decimal(2,1)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>oid</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(20)</td>
      <td>4060</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>confidence</td>
      <td></td>
      <td>(null)</td>
      <td>tinyint(1)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>url</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>60782</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>cell_id</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(20)</td>
      <td>1155</td>
      <td></td>
    </tr>
    <tr>
      <td>expression</td>
      <td>uberon_id</td>
      <td>foreign key to column uid on table uberon.</td>
      <td>UBERON:0000955</td>
      <td>varchar(20)</td>
      <td>1269</td>
      <td></td>
    </tr>
    <tr>
      <td>expression_type</td>
      <td>name</td>
      <td></td>
      <td>CCLE, Cell Surface Protein Atlas, Consensus, GTEx, HCA RNA, HPA, HPM Gene, HPM Protein, JensenLab Experiment Cardiac proteome, JensenLab Experiment Exon array, JensenLab Experiment GNF, JensenLab Experiment HPA, JensenLab Experiment HPA-RNA, JensenLab Experiment HPM, JensenLab Experiment RNA-seq, JensenLab Experiment UniGene, JensenLab Knowledge UniProtKB-RC, JensenLab Text Mining, UniProt Tissue</td>
      <td>varchar(255)</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td>expression_type</td>
      <td>data_type</td>
      <td>foreign key to column name on table data_type.</td>
      <td>integer, Boolean, String</td>
      <td>varchar(7)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>expression_type</td>
      <td>description</td>
      <td></td>
      <td>Broad Institute Cancer Cell Line Encyclopedia expression data.</td>
      <td>text</td>
      <td>18</td>
      <td>Broad Institute Cancer Cell Line Encyclopedia expression data., Cell Surface Protein Atlas protein expression in cell lines., Qualitative consensus expression value calulated from GTEx, HPA and HPM data aggregated according to manually mapped tissue types, GTEx V4 RNA-SeQCv1.1.8 Log Median RPKM and qualitative expression values per SMTSD tissue type, Human Cell Atlas gene expression in cell lines, Human Protein Atlas normal tissue expression values, Human Proteome Map gene-level Log and qualitative expression values., Human Proteome Map protein-level Log and qualitative expression values., JensenLab Experiment channel using Cardiac proteome, JensenLab Experiment channel using Exon array, JensenLab Experiment channel using GNF, JensenLab Experiment channel using Human Protein Atlas IHC, JensenLab Experiment channel using Human Protein Atlas RNA, JensenLab, Experiment channel using Humap Proteome Map, JensenLab Experiment channel using RNA-seq, JensenLab Experiment channel using UniGene, JensenLab Knowledge channel using UniProtKB-RC, JensenLab Text Mining channel, Tissue and PubMed ID from UniProt</td>
    </tr>
    <tr>
      <td>feature</td>
      <td>id</td>
      <td></td>
      <td>1- 564259</td>
      <td>int(11)</td>
      <td>564259</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>type</td>
      <td></td>
      <td>chain, binding site, modified residue, splice variant, sequence variant, helix, strand, turn, short sequence motif, sequence conflict, domain, repeat, nucleotide phosphate-binding region, region of interest, compositionally biased region, cross-link, initiator, methionine, calcium-binding region, lipid moiety-binding region, mutagenesis site, DNA-binding region, transmembrane region, coiled-coil region, active site, metal ion-binding site, site, topological domain, signal peptide, propeptide, glycosylation site, disulfide bond, zinc finger region, transit peptide, intramembrane region, peptide, non-standard amino acid, non-terminal residue, non-consecutive residues</td>
      <td>varchar(255)</td>
      <td>38</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>description</td>
      <td></td>
      <td>Cystathionine gamma-lyase</td>
      <td>text</td>
      <td>134583</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>srcid</td>
      <td></td>
      <td>PRO_0000114749</td>
      <td>varchar(255)</td>
      <td>130947</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>evidence</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>9158</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>begin</td>
      <td></td>
      <td>1</td>
      <td>int(11)</td>
      <td>5218</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>end</td>
      <td></td>
      <td>405</td>
      <td>int(11)</td>
      <td>5282</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>position</td>
      <td></td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>4940</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>original</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>feature</td>
      <td>variation</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute</td>
      <td>id</td>
      <td></td>
      <td>1-65549760</td>
      <td>int(11)</td>
      <td>65549760</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>1897</td>
      <td>int(11)</td>
      <td>18789</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute</td>
      <td>gat_id</td>
      <td>foreign key to column id on table gene_attribute_type.</td>
      <td>1</td>
      <td>int(11)</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute</td>
      <td>name</td>
      <td></td>
      <td>2-LTR circle formation/Reactome Pathways</td>
      <td>text</td>
      <td>321763</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute</td>
      <td>value</td>
      <td></td>
      <td>1</td>
      <td>int(1)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>name</td>
      <td></td>
      <td>COSMIC Cell Line Gene Mutation Profiles</td>
      <td>varchar(255)</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>association</td>
      <td></td>
      <td>gene-cell line associations by mutation of gene in cell line</td>
      <td>text</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>description</td>
      <td></td>
      <td>The Catalogue of Somatic Mutations in Cancer is a database of information about somatic mutations in cancer obtained from curation of relevant literature and from high-throughput sequencing data generated by the Cancer Genome Project and other cancer profiling projects such as The Cancer Genome Atlas.  Specifically, COSMIC collects information about point mutations, gene fusions, genomic rearrangements, and copy integer variation in cancer tissue samples and cancer cell lines.</td>
      <td>text</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>resource_group</td>
      <td></td>
      <td>genomics</td>
      <td>enum('omics','genomics','proteomics','physical interactions','transcriptomics','structural or functional annotations','disease or phenotype associations')</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>measurement</td>
      <td></td>
      <td>gene mutation by data aggregation</td>
      <td>varchar(255)</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>attribute_group</td>
      <td></td>
      <td>cell line, cell type or tissue</td>
      <td>varchar(255)</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>attribute_type</td>
      <td></td>
      <td>cell line</td>
      <td>varchar(255)</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>pubmed_ids</td>
      <td></td>
      <td>19906727|20952405</td>
      <td>text</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>gene_attribute_type</td>
      <td>url</td>
      <td></td>
      <td><a href="http://cancer.sanger.ac.uk/cancergenome/projects/cosmic/" target="_blank">http://cancer.sanger.ac.uk/cancergenome/projects/cosmic/</a></td>
      <td>text</td>
      <td>113</td>
      <td></td>
    </tr>
    <tr>
      <td>generif</td>
      <td>id</td>
      <td></td>
      <td>1-740376</td>
      <td>int(11)</td>
      <td>740376</td>
      <td></td>
    </tr>
    <tr>
      <td>generif</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>1</td>
      <td>int(11)</td>
      <td>16411</td>
      <td></td>
    </tr>
    <tr>
      <td>generif</td>
      <td>pubmed_ids</td>
      <td></td>
      <td>19692168|15151507|20634891|18701025|19625176|19324355|18676680|19161160|19913121</td>
      <td>text</td>
      <td>473814</td>
      <td></td>
    </tr>
    <tr>
      <td>generif</td>
      <td>text</td>
      <td></td>
      <td>Observational study of gene-disease association. (HuGE Navigator)</td>
      <td>text</td>
      <td>514140</td>
      <td></td>
    </tr>
    <tr>
      <td>generif</td>
      <td>years</td>
      <td></td>
      <td>2010|2004|2010|2008|2009|2010|2008|2009|2009</td>
      <td>text</td>
      <td>6041</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>id</td>
      <td></td>
      <td>1-258719</td>
      <td>int(11)</td>
      <td>258719</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19100</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>go_id</td>
      <td></td>
      <td>GO:0005737</td>
      <td>varchar(255)</td>
      <td>17903</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>go_term</td>
      <td></td>
      <td>C:cytoplasm</td>
      <td>text</td>
      <td>17903</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>evidence</td>
      <td></td>
      <td>IBA, TAS, HDA, IEA, IDA, IPI, IMP, ISS, ISM, IGI, NAS, IEP, EXP, ISA, HEP, IC, HMP, ISO</td>
      <td>text</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>goeco</td>
      <td></td>
      <td>ECO:0000318, ECO:0000304, ECO:0007005, ECO:0000501, ECO:0000314, ECO:0000353, ECO:0000315, ECO:0000250, ECO:0000255, ECO:0000316, ECO:0000303, ECO:0000270, ECO:0000269, ECO:0000247, ECO:0007007, ECO:0000305, ECO:0007001, ECO:0000266</td>
      <td>varchar(255)</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>assigned_by</td>
      <td></td>
      <td>UniProtKB</td>
      <td>varchar(50)</td>
      <td>35</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>go_type</td>
      <td></td>
      <td>Component, Function, Process</td>
      <td>enum('Component','Function','Process')</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>goa</td>
      <td>go_term_text</td>
      <td></td>
      <td>cytoplasm</td>
      <td>text</td>
      <td>17903</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>id</td>
      <td></td>
      <td>1-1805436</td>
      <td>int(11)</td>
      <td>1805436</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18474</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tissue</td>
      <td></td>
      <td>Adipose - Subcutaneous</td>
      <td>text</td>
      <td>53</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>gender</td>
      <td></td>
      <td>M, F</td>
      <td>enum('F','M')</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm</td>
      <td></td>
      <td>23.220000</td>
      <td>decimal(12,6)</td>
      <td>45047</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm_rank</td>
      <td></td>
      <td>0.653</td>
      <td>decimal(4,3)</td>
      <td>628</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm_rank_bysex</td>
      <td></td>
      <td>0.627</td>
      <td>decimal(4,3)</td>
      <td>478</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm_level</td>
      <td></td>
      <td>Low, Medium, High</td>
      <td>enum('Not detected','Low','Medium','High')</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm_level_bysex</td>
      <td></td>
      <td>Low, Medium, High</td>
      <td>enum('Not detected','Low','Medium','High')</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm_f</td>
      <td></td>
      <td>23.220000</td>
      <td>decimal(12,6)</td>
      <td>39018</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tpm_m</td>
      <td></td>
      <td>22.280000</td>
      <td>decimal(12,6)</td>
      <td>39085</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>log2foldchange</td>
      <td></td>
      <td>0.057</td>
      <td>decimal(4,3)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tau</td>
      <td></td>
      <td>0.801</td>
      <td>decimal(4,3)</td>
      <td>661</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>tau_bysex</td>
      <td></td>
      <td>0.798</td>
      <td>decimal(4,3)</td>
      <td>687</td>
      <td></td>
    </tr>
    <tr>
      <td>gtex</td>
      <td>uberon_id</td>
      <td></td>
      <td>UBERON:0002369</td>
      <td>varchar(20)</td>
      <td>48</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>id</td>
      <td></td>
      <td>1-124149</td>
      <td>int(11)</td>
      <td>124149</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>15243</td>
      <td>int(11)</td>
      <td>13116</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>disease_trait</td>
      <td></td>
      <td>Waist-to-hip ratio adjusted for BMI</td>
      <td>varchar(255)</td>
      <td>3180</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>snps</td>
      <td></td>
      <td>rs2179129</td>
      <td>text</td>
      <td>62952</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>pmid</td>
      <td></td>
      <td>26426971</td>
      <td>int(11)</td>
      <td>3230</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>study</td>
      <td></td>
      <td>The Influence of Age and Sex on Genetic Associations with Adult Body Size and Shape: A Large-Scale Genome-Wide Interaction Study.</td>
      <td>text</td>
      <td>3230</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>context</td>
      <td></td>
      <td>3_prime_UTR_variant</td>
      <td>text</td>
      <td>105</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>intergenic</td>
      <td></td>
      <td>FALSE, TRUE, (null)</td>
      <td>tinyint(1)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>p_value</td>
      <td></td>
      <td>1.00E-15</td>
      <td>double</td>
      <td>1740</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>or_beta</td>
      <td></td>
      <td>0.0266488</td>
      <td>float</td>
      <td>30345</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>cnv</td>
      <td></td>
      <td>N</td>
      <td>char(1)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>mapped_trait</td>
      <td></td>
      <td>BMI-adjusted waist-hip ratio</td>
      <td>text</td>
      <td>2693</td>
      <td></td>
    </tr>
    <tr>
      <td>gwas</td>
      <td>mapped_trait_uri</td>
      <td></td>
      <td><a href="http://www.ebi.ac.uk/efo/EFO_0007788" target="_blank">http://www.ebi.ac.uk/efo/EFO_0007788</a></td>
      <td>text</td>
      <td>2693</td>
      <td></td>
    </tr>
    <tr>
      <td>hgram_cdf</td>
      <td>id</td>
      <td></td>
      <td>1-1167880</td>
      <td>int(11)</td>
      <td>1167880</td>
      <td></td>
    </tr>
    <tr>
      <td>hgram_cdf</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18789</td>
      <td></td>
    </tr>
    <tr>
      <td>hgram_cdf</td>
      <td>type</td>
      <td>foreign key to column name on table gene_attribute_type.</td>
      <td>Allen Brain Atlas Prenatal Human Brain Tissue Gene Expression Profiles</td>
      <td>varchar(255)</td>
      <td>110</td>
      <td></td>
    </tr>
    <tr>
      <td>hgram_cdf</td>
      <td>attr_count</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2540</td>
      <td></td>
    </tr>
    <tr>
      <td>hgram_cdf</td>
      <td>attr_cdf</td>
      <td></td>
      <td>0.7703750883424460</td>
      <td>decimal(17,16)</td>
      <td>17059</td>
      <td></td>
    </tr>
    <tr>
      <td>homologgene</td>
      <td>id</td>
      <td></td>
      <td>1-69991</td>
      <td>int(11)</td>
      <td>69991</td>
      <td></td>
    </tr>
    <tr>
      <td>homologgene</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>(null), or integer</td>
      <td>int(11)</td>
      <td>18806</td>
      <td></td>
    </tr>
    <tr>
      <td>homologgene</td>
      <td>nhprotein_id</td>
      <td>foreign key to column id on table nhprotein.</td>
      <td>(null), or integer</td>
      <td>int(11)</td>
      <td>51185</td>
      <td></td>
    </tr>
    <tr>
      <td>homologgene</td>
      <td>groupid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20932</td>
      <td></td>
    </tr>
    <tr>
      <td>homologgene</td>
      <td>taxid</td>
      <td></td>
      <td>9606, 10090, 10116</td>
      <td>int(11)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>id</td>
      <td></td>
      <td>1-5118</td>
      <td>int(11)</td>
      <td>5118</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>tcrd_ver</td>
      <td></td>
      <td>TRUE</td>
      <td>tinyint(1)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>tcrd_dbid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>3900</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>name</td>
      <td></td>
      <td>5HT1A_HUMAN</td>
      <td>varchar(255)</td>
      <td>1836</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>description</td>
      <td></td>
      <td>5-hydroxytryptamine receptor 1A</td>
      <td>text</td>
      <td>1848</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>uniprot</td>
      <td></td>
      <td>P08908</td>
      <td>varchar(20)</td>
      <td>1803</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>sym</td>
      <td></td>
      <td>HTR1A</td>
      <td>varchar(20)</td>
      <td>1814</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>geneid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1806</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>tdl</td>
      <td></td>
      <td>Tclin, Tchem, Tbio, Tdark</td>
      <td>varchar(6)</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>idg_evol</td>
      <td>fam</td>
      <td></td>
      <td>GPCR, IC, Kinase, NR, oGPCR, \N</td>
      <td>varchar(20)</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_distance</td>
      <td>id</td>
      <td></td>
      <td>1-208238</td>
      <td>int(11)</td>
      <td>208238</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_distance</td>
      <td>pid1</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>4323</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_distance</td>
      <td>pid2</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>3071</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_distance</td>
      <td>distance</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_nearest_tclin</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>15911</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_nearest_tclin</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2574</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_nearest_tclin</td>
      <td>tclin_id</td>
      <td>foreign key to column id on table target\.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>403</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_nearest_tclin</td>
      <td>direction</td>
      <td></td>
      <td>upstream, downstream</td>
      <td>enum('upstream','downstream')</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>kegg_nearest_tclin</td>
      <td>distance</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td>knex_migrations</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td>knex_migrations</td>
      <td>name</td>
      <td></td>
      <td>20201222234410_createIDGtable.ts</td>
      <td>varchar(255)</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td>knex_migrations</td>
      <td>batch</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td>knex_migrations</td>
      <td>migration_time</td>
      <td></td>
      <td>2021-01-22 22:41:48</td>
      <td>timestamp</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td>knex_migrations_lock</td>
      <td>index</td>
      <td></td>
      <td>44</td>
      <td>int(10) unsigned</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>knex_migrations_lock</td>
      <td>is_locked</td>
      <td></td>
      <td>0</td>
      <td>int(11)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>lincs</td>
      <td>id</td>
      <td></td>
      <td>1-84097720</td>
      <td>int(11)</td>
      <td>84097720</td>
      <td></td>
    </tr>
    <tr>
      <td>lincs</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>980</td>
      <td></td>
    </tr>
    <tr>
      <td>lincs</td>
      <td>cellid</td>
      <td></td>
      <td>HELA</td>
      <td>varchar(10)</td>
      <td>82</td>
      <td></td>
    </tr>
    <tr>
      <td>lincs</td>
      <td>zscore</td>
      <td></td>
      <td>-0.436812</td>
      <td>decimal(8,6)</td>
      <td>7804415</td>
      <td></td>
    </tr>
    <tr>
      <td>lincs</td>
      <td>pert_dcid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1613</td>
      <td></td>
    </tr>
    <tr>
      <td>lincs</td>
      <td>pert_smiles</td>
      <td></td>
      <td>OC(=O)CCc1nc(c(o1)-c1ccccc1)-c1ccccc1</td>
      <td>text</td>
      <td>2147</td>
      <td></td>
    </tr>
    <tr>
      <td>locsig</td>
      <td>id</td>
      <td></td>
      <td>1-106521</td>
      <td>int(11)</td>
      <td>106521</td>
      <td></td>
    </tr>
    <tr>
      <td>locsig</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18916</td>
      <td></td>
    </tr>
    <tr>
      <td>locsig</td>
      <td>location</td>
      <td></td>
      <td>Lysosome|Melanosome</td>
      <td>varchar(255)</td>
      <td>14</td>
      <td></td>
    </tr>
    <tr>
      <td>locsig</td>
      <td>signal</td>
      <td></td>
      <td>[DE]x{3}L[LI]</td>
      <td>varchar(255)</td>
      <td>31</td>
      <td></td>
    </tr>
    <tr>
      <td>locsig</td>
      <td>pmids</td>
      <td></td>
      <td>19116314|16262729|19523115|18479248|17635580</td>
      <td>text</td>
      <td>239</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>id</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>assay_name</td>
      <td></td>
      <td></td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>method</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>active_sids</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>inactive_sids</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>iconclusive_sids</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>total_sids</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mlp_assay_info</td>
      <td>aid</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>mondo?</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>mpo</td>
      <td>mpid</td>
      <td></td>
      <td>MP:0000003</td>
      <td>char(10)</td>
      <td>12894</td>
      <td></td>
    </tr>
    <tr>
      <td>mpo</td>
      <td>parent_id</td>
      <td></td>
      <td>MP:0005375</td>
      <td>varchar(10)</td>
      <td>4283</td>
      <td></td>
    </tr>
    <tr>
      <td>mpo</td>
      <td>name</td>
      <td></td>
      <td>abnormal adipose tissue morphology</td>
      <td>text</td>
      <td>12894</td>
      <td></td>
    </tr>
    <tr>
      <td>mpo</td>
      <td>def</td>
      <td></td>
      <td>any structural anomaly of the connective tissue composed of fat cells enmeshed in areolar tissue</td>
      <td>text</td>
      <td>12689</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>1301090</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>dataSource</td>
      <td></td>
      <td>Antibodypedia</td>
      <td>varchar(50)</td>
      <td>78</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>url</td>
      <td></td>
      <td><a href="https://www.antibodypedia.com/" target="_blank">https://www.antibodypedia.com/</a></td>
      <td>varchar(128)</td>
      <td>57</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>license</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(128)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>licenseURL</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(128)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>protein_id</td>
      <td></td>
      <td>1-20412</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>ncats_ligand_id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>266944</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_dataSource_map</td>
      <td>disease_name</td>
      <td></td>
      <td>schizophrenia</td>
      <td>text</td>
      <td>24328</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_do</td>
      <td>lft</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>29191</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_do</td>
      <td>rght</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>23109</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_do</td>
      <td>doid</td>
      <td></td>
      <td>DOID:223</td>
      <td>varchar(12)</td>
      <td>9233</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_do</td>
      <td>name</td>
      <td></td>
      <td>ulnar-mammary syndrome</td>
      <td>text</td>
      <td>23109</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_do</td>
      <td>def</td>
      <td></td>
      <td>A syndrome that has_material_basis_in heterozygous mutation in the TBX3 gene. It is characterized by posterior limb deficiencies or duplications, apocrine/mammary gland hypoplasia and, or dysfunction, abnormal dentition, delayed puberty in males, and genital anomalies.</td>
      <td>text</td>
      <td>6119</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_facet_expression</td>
      <td>etype</td>
      <td></td>
      <td>HPM Protein</td>
      <td>varchar(255)</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_facet_expression</td>
      <td>name</td>
      <td></td>
      <td>Adult Heart</td>
      <td>text</td>
      <td>5720</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_facet_expression</td>
      <td>value</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>624</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_facet_impc</td>
      <td>name</td>
      <td></td>
      <td>abnormal auditory brainstem response</td>
      <td>varchar(255)</td>
      <td>750</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_facet_impc</td>
      <td>value</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>151</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_generif_pubmed_map</td>
      <td>generif_id</td>
      <td></td>
      <td>138</td>
      <td>int(11)</td>
      <td>740376</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_generif_pubmed_map</td>
      <td>pubmed_id</td>
      <td></td>
      <td>25681036</td>
      <td>int(11)</td>
      <td>491623</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list</td>
      <td>id</td>
      <td></td>
      <td>1-1068</td>
      <td>int(10) unsigned</td>
      <td>1068</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list</td>
      <td>protein_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>417</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list</td>
      <td>symbol</td>
      <td></td>
      <td>STK40</td>
      <td>varchar(20)</td>
      <td>417</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list</td>
      <td>family</td>
      <td></td>
      <td>GPCR, Kinase, Ion Channel</td>
      <td>varchar(20)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list</td>
      <td>idg_list</td>
      <td></td>
      <td>1,2,3</td>
      <td>int(10) unsigned</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list_type</td>
      <td>id</td>
      <td></td>
      <td>1,2,3</td>
      <td>int(10) unsigned</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_idg_list_type</td>
      <td>list_type</td>
      <td></td>
      <td>Original (2017) Refined (2019) Refined (2020)</td>
      <td>varchar(255)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>id</td>
      <td></td>
      <td>1-504997</td>
      <td>int(10) unsigned</td>
      <td>504997</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>ncats_ligand_id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>266944</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>target_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2266</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>smiles</td>
      <td></td>
      <td>CC(=O)Nc1nnc(s1)S(=O)(=O)N</td>
      <td>text</td>
      <td>271031</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>act_value</td>
      <td></td>
      <td>float integer</td>
      <td>decimal(10,8)</td>
      <td>1827</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>act_type</td>
      <td></td>
      <td>AC50</td>
      <td>varchar(255)</td>
      <td>15</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>action_type</td>
      <td></td>
      <td>PARTIAL AGONIST</td>
      <td>varchar(255)</td>
      <td>22</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>reference</td>
      <td></td>
      <td>Bioorg. Med. Chem. Lett., (2008) 18:6:1898</td>
      <td>text</td>
      <td>17841</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>reference_source</td>
      <td></td>
      <td>SCIENTIFIC LITERATURE</td>
      <td>varchar(255)</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligand_activity</td>
      <td>pubmed_ids</td>
      <td></td>
      <td>18295485</td>
      <td>text</td>
      <td>18504</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>id</td>
      <td></td>
      <td>1-266764</td>
      <td>int(10) unsigned</td>
      <td>266944</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>identifier</td>
      <td></td>
      <td>T5X39HC6G9BK</td>
      <td>varchar(255)</td>
      <td>266944</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>name</td>
      <td></td>
      <td>lutetium (177Lu) oxodotreotide</td>
      <td>text</td>
      <td>265320</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>isDrug</td>
      <td></td>
      <td>boolean</td>
      <td>tinyint(1)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>smiles</td>
      <td></td>
      <td>[177Lu].C[C@@H](O)[C@H](NC(=O)C1CSSC[C@H](NC(=O)C(CC2=CC=CC=C2)NC(=O)CN2CCN(CC(O)=O)CCN(CC(O)=O)CCN(CC(O)=O)CC2)C(=O)N[C@@H](CC2=CC=C(O)C=C2)C(=O)N[C@H](CC2=CNC3=C2C=CC=C3)C(=O)N[C@@H](CCCCN)C(=O)N[C@@H]([C@@H](C)O)C(=O)N1)C(O)=O</td>
      <td>text</td>
      <td>266764</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>PubChem</td>
      <td></td>
      <td>integer</td>
      <td>varchar(255)</td>
      <td>247250</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>ChEMBL</td>
      <td></td>
      <td>CHEMBL3989924</td>
      <td>varchar(255)</td>
      <td>264326</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>Guide to Pharmacology</td>
      <td></td>
      <td>integer</td>
      <td>varchar(255)</td>
      <td>1640</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>DrugCentral</td>
      <td></td>
      <td>integer</td>
      <td>varchar(255)</td>
      <td>1640</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>description</td>
      <td></td>
      <td>Lutetium Lu 177 dotatate binds to somatostatin receptors with highest affinity for subtype 2 receptors (SSTR2). Upon binding to somatostatin receptor expressing cells, including malignant somatostatin receptor-positive tumors, the compound is internalized. The beta emission from Lu 177 induces cellular damage by formation of free radicals in somatostatin receptor-positive cells and in neighboring cells.</td>
      <td>text</td>
      <td>1471</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ligands</td>
      <td>actCnt</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>11672660</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>ppitypes</td>
      <td></td>
      <td>STRINGDB BioPlex BioPlex,STRINGDB Reactome,STRINGDB BioPlex,Reactome,STRINGDB Reactome BioPlex,Reactome</td>
      <td>varchar(255)</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>protein_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19165</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>other_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19165</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>p_int</td>
      <td></td>
      <td>float</td>
      <td>decimal(10,9)</td>
      <td>68502</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>p_ni</td>
      <td></td>
      <td>float</td>
      <td>decimal(10,9)</td>
      <td>67323</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>p_wrong</td>
      <td></td>
      <td>float</td>
      <td>decimal(10,9)</td>
      <td>18874</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>evidence</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>interaction_type</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(100)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_ppi</td>
      <td>score</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>850</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_typeahead_index</td>
      <td>id</td>
      <td></td>
      <td>1-86764</td>
      <td>int(10) unsigned</td>
      <td>86764</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_typeahead_index</td>
      <td>category</td>
      <td></td>
      <td>Diseases</td>
      <td>varchar(45)</td>
      <td>15</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_typeahead_index</td>
      <td>value</td>
      <td></td>
      <td>2-methylbutyryl-CoA dehydrogenase deficiency</td>
      <td>varchar(255)</td>
      <td>86019</td>
      <td></td>
    </tr>
    <tr>
      <td>ncats_typeahead_index</td>
      <td>reference_id</td>
      <td></td>
      <td>2-methylbutyryl-CoA dehydrogenase deficiency</td>
      <td>varchar(255)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>id</td>
      <td></td>
      <td>1- 121277</td>
      <td>int(11)</td>
      <td>121277</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>uniprot</td>
      <td></td>
      <td>Q9CQA6</td>
      <td>varchar(20)</td>
      <td>121277</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>name</td>
      <td></td>
      <td>CHCH1_MOUSE</td>
      <td>varchar(255)</td>
      <td>121277</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>description</td>
      <td></td>
      <td>Coiled-coil-helix-coiled-coil-helix domain-containing protein 1</td>
      <td>text</td>
      <td>47393</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>sym</td>
      <td></td>
      <td>Chchd1</td>
      <td>varchar(30)</td>
      <td>29381</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>species</td>
      <td></td>
      <td>Rattus norvegicus Mus musculus</td>
      <td>varchar(40)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>taxid</td>
      <td></td>
      <td>10090, 10116</td>
      <td>int(11)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>nhprotein</td>
      <td>geneid</td>
      <td></td>
      <td>66121</td>
      <td>int(11)</td>
      <td>38094</td>
      <td></td>
    </tr>
    <tr>
      <td>omim</td>
      <td>mim</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>26221</td>
      <td></td>
    </tr>
    <tr>
      <td>omim</td>
      <td>title</td>
      <td></td>
      <td>AARSKOG SYNDROME, AUTOSOMAL DOMINANT</td>
      <td>varchar(255)</td>
      <td>25960</td>
      <td></td>
    </tr>
    <tr>
      <td>omim_ps</td>
      <td>id</td>
      <td></td>
      <td>1-4375</td>
      <td>int(11)</td>
      <td>4375</td>
      <td></td>
    </tr>
    <tr>
      <td>omim_ps</td>
      <td>omim_ps_id</td>
      <td></td>
      <td>PS100070</td>
      <td>char(8)</td>
      <td>450</td>
      <td></td>
    </tr>
    <tr>
      <td>omim_ps</td>
      <td>mim</td>
      <td>foreign key to column mim on table omim.</td>
      <td>100070</td>
      <td>int(11)</td>
      <td>3628</td>
      <td></td>
    </tr>
    <tr>
      <td>omim_ps</td>
      <td>title</td>
      <td></td>
      <td>Aortic aneurysm, familial abdominal 1</td>
      <td>varchar(255)</td>
      <td>4232</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>178759</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>1-18056</td>
      <td>int(11)</td>
      <td>18056</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>taxid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>17</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>species</td>
      <td></td>
      <td>Cow</td>
      <td>varchar(255)</td>
      <td>17</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>db_id</td>
      <td></td>
      <td>VGNC:27795</td>
      <td>varchar(255)</td>
      <td>113894</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>geneid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>169587</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>symbol</td>
      <td></td>
      <td>CTH</td>
      <td>varchar(255)</td>
      <td>33498</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>name</td>
      <td></td>
      <td>cystathionine gamma-lyase</td>
      <td>varchar(255)</td>
      <td>49376</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>mod_url</td>
      <td></td>
      <td><a href="http://www.informatics.jax.org/marker/MGI:1339968" target="_blank">http://www.informatics.jax.org/marker/MGI:1339968</a></td>
      <td>text</td>
      <td>49093</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog</td>
      <td>sources</td>
      <td></td>
      <td>OMA, EggNOG Inparanoid, OMA, EggNOG Inparanoid, OMA Inparanoid, EggNOG</td>
      <td>varchar(255)</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog_disease</td>
      <td>id</td>
      <td></td>
      <td>1-37852</td>
      <td>int(11)</td>
      <td>37852</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog_disease</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>3827</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog_disease</td>
      <td>did</td>
      <td></td>
      <td>OMIM:125460</td>
      <td>varchar(255)</td>
      <td>5641</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog_disease</td>
      <td>name</td>
      <td></td>
      <td>Deoxyribose-5-Phosphate Aldolase Deficiency</td>
      <td>varchar(255)</td>
      <td>5641</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog_disease</td>
      <td>ortholog_id</td>
      <td>foreign key to column id on table ortholog.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>3827</td>
      <td></td>
    </tr>
    <tr>
      <td>ortholog_disease</td>
      <td>score</td>
      <td></td>
      <td>float</td>
      <td>varchar(255)</td>
      <td>23678</td>
      <td></td>
    </tr>
    <tr>
      <td>p2dto</td>
      <td>dtoid</td>
      <td></td>
      <td>CHEBI:34972</td>
      <td>varchar(255)</td>
      <td>10101</td>
      <td></td>
    </tr>
    <tr>
      <td>p2dto</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>9138</td>
      <td></td>
    </tr>
    <tr>
      <td>p2dto</td>
      <td>generation</td>
      <td></td>
      <td>0,1,2,3,4,5,6,7</td>
      <td>int(11)</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td>p2pc</td>
      <td>panther_class_id</td>
      <td>foreign key to column id on table panther_class.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>214</td>
      <td></td>
    </tr>
    <tr>
      <td>p2pc</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>8070</td>
      <td></td>
    </tr>
    <tr>
      <td>panther_class</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>256</td>
      <td></td>
    </tr>
    <tr>
      <td>panther_class</td>
      <td>pcid</td>
      <td></td>
      <td>PC00035</td>
      <td>char(7)</td>
      <td>256</td>
      <td></td>
    </tr>
    <tr>
      <td>panther_class</td>
      <td>parent_pcids</td>
      <td></td>
      <td>PC00084</td>
      <td>varchar(255)</td>
      <td>79</td>
      <td></td>
    </tr>
    <tr>
      <td>panther_class</td>
      <td>name</td>
      <td></td>
      <td>TGF-beta receptor</td>
      <td>text</td>
      <td>256</td>
      <td></td>
    </tr>
    <tr>
      <td>panther_class</td>
      <td>description</td>
      <td></td>
      <td></td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>patent_count</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>41280</td>
      <td></td>
    </tr>
    <tr>
      <td>patent_count</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1710</td>
      <td></td>
    </tr>
    <tr>
      <td>patent_count</td>
      <td>year</td>
      <td></td>
      <td>2016</td>
      <td>int(4)</td>
      <td>44</td>
      <td></td>
    </tr>
    <tr>
      <td>patent_count</td>
      <td>count</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>4650</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>id</td>
      <td></td>
      <td>1-331936</td>
      <td>int(11)</td>
      <td>331936</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>12984</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>pwtype</td>
      <td>foreign key to column name on table pathway_type.</td>
      <td>KEGG, PathwayCommons: humancyc, PathwayCommons: inoh, PathwayCommons: netpath, PathwayCommons: panther, PathwayCommons: pid, Reactome, UniProt, WikiPathways</td>
      <td>varchar(255)</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>id_in_source</td>
      <td></td>
      <td>path:hsa04151</td>
      <td>varchar(255)</td>
      <td>3109</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>name</td>
      <td></td>
      <td>PI3K-Akt signaling pathway</td>
      <td>text</td>
      <td>4516</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>description</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway</td>
      <td>url</td>
      <td></td>
      <td><a href="http://www.kegg.jp/kegg-bin/show_pathway?hsa04151" target="_blank">http://www.kegg.jp/kegg-bin/show_pathway?hsa04151</a></td>
      <td>text</td>
      <td>4323</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway_type</td>
      <td>name</td>
      <td></td>
      <td>KEGG PathwayCommons: ctd PathwayCommons: humancyc PathwayCommons: inoh PathwayCommons: mirtarbase PathwayCommons: netpath PathwayCommons: panther PathwayCommons: pid PathwayCommons: recon PathwayCommons: smpdb PathwayCommons: transfac Reactome UniProt WikiPathways</td>
      <td>varchar(255)</td>
      <td>14</td>
      <td></td>
    </tr>
    <tr>
      <td>pathway_type</td>
      <td>url</td>
      <td></td>
      <td><a href="http://www.kegg.jp/kegg/pathway.html http://www.pathwaycommons.org/pc2/ctd http://www.pathwaycommons.org/pc2/humancyc http://www.pathwaycommons.org/pc2/inoh http://www.pathwaycommons.org/pc2/mirtarbase http://www.pathwaycommons.org/pc2/netpath http://pathwaycommons.org/pc2/panther http://www.pathwaycommons.org/pc2/pid http://www.pathwaycommons.org/pc2/recon http://www.pathwaycommons.org/pc2/smpdb http://www.pathwaycommons.org/pc2/transfac http://www.reactome.org/ http://www.uniprot.org/ http://www.wikipathways.org/index.php/WikiPathways" target="_blank">http://www.kegg.jp/kegg/pathway.html http://www.pathwaycommons.org/pc2/ctd http://www.pathwaycommons.org/pc2/humancyc http://www.pathwaycommons.org/pc2/inoh http://www.pathwaycommons.org/pc2/mirtarbase http://www.pathwaycommons.org/pc2/netpath http://pathwaycommons.org/pc2/panther http://www.pathwaycommons.org/pc2/pid http://www.pathwaycommons.org/pc2/recon http://www.pathwaycommons.org/pc2/smpdb http://www.pathwaycommons.org/pc2/transfac http://www.reactome.org/ http://www.uniprot.org/ http://www.wikipathways.org/index.php/WikiPathways</a></td>
      <td>text</td>
      <td>14</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>7318480</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>ptype</td>
      <td>foreign key to column name on table phenotype_type.</td>
      <td>IMPC, JAX/MGI Human Ortholog Phenotype, OMIM</td>
      <td>varchar(255)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>15397</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>nhprotein_id</td>
      <td>foreign key to column id on table nhprotein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>27637</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>trait</td>
      <td></td>
      <td>MIM integer: 103320; Phenotype: Myasthenic syndrome, congenital, 8, with pre- and postsynaptic defects, 615120 (3)</td>
      <td>text</td>
      <td>13751</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>top_level_term_id</td>
      <td></td>
      <td>MP:0005378</td>
      <td>varchar(255)</td>
      <td>76</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>top_level_term_name</td>
      <td></td>
      <td>cardiovascular system phenotype</td>
      <td>varchar(255)</td>
      <td>76</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>term_id</td>
      <td></td>
      <td>MP:0005386</td>
      <td>varchar(255)</td>
      <td>738</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>term_name</td>
      <td></td>
      <td>behavior/neurological phenotype</td>
      <td>varchar(255)</td>
      <td>741</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>term_description</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>p_value</td>
      <td></td>
      <td>float</td>
      <td>double</td>
      <td>452786</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>percentage_change</td>
      <td></td>
      <td>-186.1220193</td>
      <td>varchar(255)</td>
      <td>7282</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>effect_size</td>
      <td></td>
      <td>float</td>
      <td>varchar(255)</td>
      <td>11149</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>procedure_name</td>
      <td></td>
      <td>Intraperitoneal glucose tolerance test (IPGTT)</td>
      <td>varchar(255)</td>
      <td>92</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>parameter_name</td>
      <td></td>
      <td>Fat mass</td>
      <td>varchar(255)</td>
      <td>967</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>gp_assoc</td>
      <td></td>
      <td>boolean</td>
      <td>tinyint(1)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>statistical_method</td>
      <td></td>
      <td>(null) Mixed Model framework, linear mixed-effects model, equation withoutWeight Mixed Model framework, generalized least squares, equation withWeight Mixed Model framework, generalized least squares, equation withoutWeight Mixed Model framework, linear mixed-effects model, equation withWeight Unknown Fisher Exact Test framework Supplied as data Not processed  Manual</td>
      <td>text</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype</td>
      <td>sex</td>
      <td></td>
      <td>(null), female, male, no_data, both</td>
      <td>varchar(8)</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype_type</td>
      <td>name</td>
      <td></td>
      <td>GWAS Catalog IMPC JAX/MGI Human Ortholog Phenotype OMIM</td>
      <td>varchar(255)</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype_type</td>
      <td>ontology</td>
      <td></td>
      <td>(null) Mammalian Phenotype Ontology Mammalian Phenotype Ontology (null)</td>
      <td>varchar(255)</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>phenotype_type</td>
      <td>description</td>
      <td></td>
      <td>GWAS findings from NHGRI/EBI GWAS catalog file. Phenotypes from the International Mouse Phenotyping Consortium. These are single gene knockout phenotypes. JAX/MGI house/human orthology phenotypes in file HMD_HumanPhenotype.rpt from ftp.informatics.jax.or Phenotypes from OMIM with status Confirmed. phenotype.trait is a concatenation of Title, MIM integer, Method, and Comments fields from the OMIM genemap2.txt file.</td>
      <td>text</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td>pmscore</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>384954</td>
      <td></td>
    </tr>
    <tr>
      <td>pmscore</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>17982</td>
      <td></td>
    </tr>
    <tr>
      <td>pmscore</td>
      <td>year</td>
      <td></td>
      <td>integer</td>
      <td>int(4)</td>
      <td>119</td>
      <td></td>
    </tr>
    <tr>
      <td>pmscore</td>
      <td>score</td>
      <td></td>
      <td>float</td>
      <td>decimal(12,6)</td>
      <td>168075</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>11735789</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>ppitype</td>
      <td></td>
      <td>BioPlex Reactome STRINGDB</td>
      <td>varchar(255)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>protein1_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19113</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>protein1_str</td>
      <td></td>
      <td>P62136|PPP1CA|5499</td>
      <td>varchar(255)</td>
      <td>28935</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>protein2_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19136</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>protein2_str</td>
      <td></td>
      <td>Q9ULJ8|PPP1R9A|55607</td>
      <td>varchar(255)</td>
      <td>36977</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>p_int</td>
      <td></td>
      <td>Q07021|C1QBP|708</td>
      <td>decimal(10,9)</td>
      <td>69661</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>p_ni</td>
      <td></td>
      <td>float</td>
      <td>decimal(10,9)</td>
      <td>68441</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>p_wrong</td>
      <td></td>
      <td>float</td>
      <td>decimal(10,9)</td>
      <td>18917</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>evidence</td>
      <td></td>
      <td>0</td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>interaction_type</td>
      <td></td>
      <td>(null)</td>
      <td>varchar(100)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi</td>
      <td>score</td>
      <td></td>
      <td>integer</td>
      <td>int(4)</td>
      <td>850</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi_type</td>
      <td>name</td>
      <td></td>
      <td>BioPlex Reactome</td>
      <td>varchar(255)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi_type</td>
      <td>description</td>
      <td></td>
      <td>The BioPlex (biophysical interactions of ORFeome-based complexes) network is the result of creating thousands of cell lines with each expressing a tagged version of a protein from the ORFeome collection. Interactions derived from Reactome pathways</td>
      <td>text</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>ppi_type</td>
      <td>url</td>
      <td></td>
      <td><a href="http://wren.hms.harvard.edu/bioplex/ http://www.reactome.org/" target="_blank">http://wren.hms.harvard.edu/bioplex/ http://www.reactome.org/</a></td>
      <td>text</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>id</td>
      <td>TCRD protein ID</td>
      <td>1 - 20412</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>name</td>
      <td></td>
      <td>CHERP_HUMAN</td>
      <td>varchar(255)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>description</td>
      <td></td>
      <td>Calcium homeostasis endoplasmic reticulum protein</td>
      <td>text</td>
      <td>20405</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>uniprot</td>
      <td>UniProt ID</td>
      <td>Q8IWX8</td>
      <td>varchar(20)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>up_version</td>
      <td>UniProt ID version</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>geneid</td>
      <td>NCBI gene ID</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20011</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>sym</td>
      <td>Protein symbol</td>
      <td>CHP1</td>
      <td>varchar(20)</td>
      <td>20080</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>family</td>
      <td></td>
      <td>Belongs to the calcineurin regulatory subunit family. CHP subfamily.</td>
      <td>varchar(255)</td>
      <td>5043</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>chr</td>
      <td>chromosome</td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>seq</td>
      <td></td>
      <td>MQEKDASSQGFLPHFQHFATQAIHVGQDPEQWTSRAVVPPISLSTTFKQGAPGQHSGFEYSRSGNPTRNCLEKAVAALDGAKYCLAFASGLAATVTITHLLKAGDQIICMDDVYGGTNRYFRQVASEFGLKISFVDCSKIKLLEAAITPETKLVWIETPTNPTQKVIDIEGCAHIVHKHGDIILVVDNTFMSPYFQRPLALGADISMYSATKYMNGHSDVVMGLVSVNCESLHNRLRFLQNSLGAVPSPIDCYLCNRGLKTLHVRMEKHFKNGMAVAQFLESNPWVEKVIYPGLPSHPQHELVKRQCTGCTGMVTFYIKGTLQHAEIFLKNLKLFTLAESLGGFESLAELPAIMTHASVLKNDRDVLGISDTLIRLSVGLEDEEDLLEDLDQALKAAHPPSGSHS</td>
      <td>text</td>
      <td>20348</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>dtoid</td>
      <td>DTO ID</td>
      <td>DTO_05004572</td>
      <td>varchar(13)</td>
      <td>9232</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>stringid</td>
      <td>Ensembl protein ID (as used by STRING-DB)</td>
      <td>ENSP00000359976</td>
      <td>varchar(15)</td>
      <td>18960</td>
      <td></td>
    </tr>
    <tr>
      <td>protein</td>
      <td>dtoclass</td>
      <td>DTO class</td>
      <td>Lyase</td>
      <td>varchar(255)</td>
      <td>860</td>
      <td></td>
    </tr>
    <tr>
      <td>protein2pubmed</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>19790</td>
      <td></td>
    </tr>
    <tr>
      <td>protein2pubmed</td>
      <td>pubmed_id</td>
      <td>foreign key to column id on table pubmed</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>576647</td>
      <td></td>
    </tr>
    <tr>
      <td>provenance</td>
      <td></td>
      <td>see datasets tab</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>ptscore</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>468019</td>
      <td></td>
    </tr>
    <tr>
      <td>ptscore</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18310</td>
      <td></td>
    </tr>
    <tr>
      <td>ptscore</td>
      <td>year</td>
      <td></td>
      <td>integer</td>
      <td>int(4)</td>
      <td>119</td>
      <td></td>
    </tr>
    <tr>
      <td>ptscore</td>
      <td>score</td>
      <td></td>
      <td>float</td>
      <td>decimal(12,6)</td>
      <td>59549</td>
      <td></td>
    </tr>
    <tr>
      <td>pubmed</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2746858</td>
      <td></td>
    </tr>
    <tr>
      <td>pubmed</td>
      <td>title</td>
      <td></td>
      <td>Aflatoxin B1 metabolism to aflatoxicol and derivatives lethal to Bacillus subtilis GSY 1057 by rainbow trout (Salmo gairdneri) liver.</td>
      <td>text</td>
      <td>2737912</td>
      <td></td>
    </tr>
    <tr>
      <td>pubmed</td>
      <td>journal</td>
      <td></td>
      <td>Cancer research</td>
      <td>text</td>
      <td>12234</td>
      <td></td>
    </tr>
    <tr>
      <td>pubmed</td>
      <td>date</td>
      <td></td>
      <td>1976-06</td>
      <td>varchar(10)</td>
      <td>18197</td>
      <td></td>
    </tr>
    <tr>
      <td>pubmed</td>
      <td>authors</td>
      <td></td>
      <td>Schoenhard, G L GL and 5 more authors.</td>
      <td>text</td>
      <td>2423527</td>
      <td></td>
    </tr>
    <tr>
      <td>pubmed</td>
      <td>abstract</td>
      <td></td>
      <td>Aflatoxicol, R0, was isolated from Mt. Shasta strain rainbow trout (Salmo gairdneri), and liver homogenates were incubated with aflatoxin B1. Its identity was confirmed by mass, infrared, and ultraviolet spectrometry. The structure was identical to one of the diastereomers prepared by chemical reduction of aflatoxin B1. Aflatoxicol was apparently formed by a reduced nicotinamide adenine dinucleotide phosphate-dependent soluble enzyme of the 105,000 x g supernatant from rainbow trout. Aflatoxicol was not lethal in phosphate buffer to Bacillus subtilis GSY 1057 (metB4, hisA1, uvr-1) nor were aflatoxins B1, Q1, and B2. In the presence of reduced nicotinamide adenine dinucleotide phosphate and trout liver microsomes, aflatoxicol reduced the viability of B. subtilis. Aflatoxin B2, which lacks the vinyl ether present in the other compounds, could not be activated. The product of aflatoxin B1 activation by trout liver microsomes was sought after incubation of 14C-labeled aflatoxin B1. The radioactivity was found in unaltered aflatoxin B1 and in three extremely polar metabolites. The quantity of the new metabolites and the level of microbial lethality was reduced by addition of cytosine and cysteine to the incubation medium. The vinyl ether configuration was a structural requirement for activation, and this finding and the nature of the enzymatic reaction were consistent with the hypothesis that the compounds were metabolized to highly reactive and unstable electrophilic products which bound to nucleophiles such as cytosine and were lethal to B. subtilis. The formation of aflatoxicol as the major product of trout liver metabolism is of great significance considering that it could be activated to a lethal compound and that rainbow trout are one of the most sensitive species to aflatoxin B1-induced carcinoma.</td>
      <td>text</td>
      <td>2666654</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1799</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>nhprotein_id</td>
      <td>foreign key to column id on table nhprotein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>908</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>rgdid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>502</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>qtl_rgdid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>301</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>qtl_symbol</td>
      <td></td>
      <td>Plsm2</td>
      <td>varchar(10)</td>
      <td>301</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>qtl_name</td>
      <td></td>
      <td>Polydactyly-luxate syndrome (PLS) morphotypes QTL 2</td>
      <td>varchar(255)</td>
      <td>301</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>trait_name</td>
      <td></td>
      <td>hindlimb integrity trait (VT:0010563)</td>
      <td>varchar(255)</td>
      <td>74</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>measurement_type</td>
      <td></td>
      <td>hind foot phalanges count (CMO:0001949)</td>
      <td>varchar(255)</td>
      <td>109</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>associated_disease</td>
      <td></td>
      <td>polydactyly</td>
      <td>varchar(255)</td>
      <td>82</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>phenotype</td>
      <td></td>
      <td>polydactyly</td>
      <td>varchar(255)</td>
      <td>126</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>p_value</td>
      <td></td>
      <td>float</td>
      <td>decimal(20,19)</td>
      <td>44</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_qtl</td>
      <td>lod</td>
      <td></td>
      <td>float</td>
      <td>float(6,3)</td>
      <td>114</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>117602</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>rgdid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>10127</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>term_id</td>
      <td></td>
      <td>DOID:1612</td>
      <td>varchar(20)</td>
      <td>11332</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>obj_symbol</td>
      <td></td>
      <td>Six1</td>
      <td>varchar(20)</td>
      <td>10127</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>term_name</td>
      <td></td>
      <td>breast cancer</td>
      <td>varchar(255)</td>
      <td>10058</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>qualifier</td>
      <td></td>
      <td>severity</td>
      <td>varchar(20)</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>evidence</td>
      <td></td>
      <td>ISS</td>
      <td>varchar(5)</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td>rat_term</td>
      <td>ontology</td>
      <td></td>
      <td>Disease Ontology, Mammalian Phenotype, RGD Disease Ontology</td>
      <td>varchar(40)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>rdo</td>
      <td>doid</td>
      <td></td>
      <td>DOID:0001816</td>
      <td>varchar(12)</td>
      <td>18085</td>
      <td></td>
    </tr>
    <tr>
      <td>rdo</td>
      <td>name</td>
      <td></td>
      <td>angiosarcoma</td>
      <td>text</td>
      <td>18085</td>
      <td></td>
    </tr>
    <tr>
      <td>rdo</td>
      <td>def</td>
      <td></td>
      <td>A rare malignant neoplasm characterized by rapidly proliferating, extensively infiltrating, anaplastic cells derived from blood vessels and lining irregular blood-filled or lumpy spaces. (Stedman, 25th ed)</td>
      <td>text</td>
      <td>10034</td>
      <td></td>
    </tr>
    <tr>
      <td>rdo_xref</td>
      <td>doid</td>
      <td>foreign key to column doid on table rdo.</td>
      <td>DOID:9009161</td>
      <td>varchar(12)</td>
      <td>17318</td>
      <td></td>
    </tr>
    <tr>
      <td>rdo_xref</td>
      <td>db</td>
      <td></td>
      <td>DOID</td>
      <td>varchar(24)</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td>rdo_xref</td>
      <td>value</td>
      <td></td>
      <td>number</td>
      <td>varchar(24)</td>
      <td>38831</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>3097</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>input</td>
      <td></td>
      <td>alcaftadine</td>
      <td>text</td>
      <td>3097</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>smilesParent</td>
      <td></td>
      <td>CN1CCC(CC1)=C2C3=NC=C(C=O)N3CCC4=C2C=CC=C4</td>
      <td>text</td>
      <td>2000</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>pt</td>
      <td></td>
      <td>ALCAFTADINE</td>
      <td>varchar(255)</td>
      <td>1788</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>unii</td>
      <td></td>
      <td>7Z8O94ECSX</td>
      <td>varchar(12)</td>
      <td>1704</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>smiles</td>
      <td></td>
      <td>CN1CCC(CC1)=C2C3=NC=C(C=O)N3CCC4=C2C=CC=C4</td>
      <td>text</td>
      <td>2033</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>lychi</td>
      <td></td>
      <td>S4GHBD1FU-UDYRYTP1VB-UBC734PAGQ9-UB9SA8WABQD2</td>
      <td>varchar(50)</td>
      <td>1528</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>lychi_h4</td>
      <td></td>
      <td>UB9SA8WABQD2</td>
      <td>varchar(15)</td>
      <td>1528</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>cas</td>
      <td></td>
      <td>147084-10-4</td>
      <td>varchar(255)</td>
      <td>1787</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>method</td>
      <td></td>
      <td>FDA-SRS null (null) smiles</td>
      <td>varchar(255)</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td>resolver_output</td>
      <td>url</td>
      <td></td>
      <td><a href="http://fdasis.nlm.nih.gov/srs/srsdirect.jsp?regno=7Z8O94ECSX" target="_blank">http://fdasis.nlm.nih.gov/srs/srsdirect.jsp?regno=7Z8O94ECSX</a></td>
      <td>varchar(255)</td>
      <td>1437</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>32519</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>dataSource</td>
      <td></td>
      <td>ProKinO</td>
      <td>varchar(255)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>protein_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>472</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>residue_start</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1883</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>residue_end</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2000</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>type</td>
      <td></td>
      <td>Glycine Loop</td>
      <td>enum('Activation Loop','Activation Segment','alphaC-beta4 Loop','CMGC Insert','Gatekeeper','Linker','KeyAA','Motif','beta-strand','alpha-helix','C-Lobe','C-Spine','RD Pocket','Catalytic Loop','Glycine Loop','N-Lobe','R-Spine','R-Spine Shell','Subdomain')</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_annotation</td>
      <td>name</td>
      <td></td>
      <td>glycine-rich loop</td>
      <td>varchar(255)</td>
      <td>70</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_variant</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(10) unsigned</td>
      <td>1063403</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_variant</td>
      <td>dataSource</td>
      <td></td>
      <td>ProKinO</td>
      <td>varchar(255)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_variant</td>
      <td>protein_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>543</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_variant</td>
      <td>residue</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>34350</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_variant</td>
      <td>variant</td>
      <td></td>
      <td>amino acid letter</td>
      <td>varchar(1)</td>
      <td>20</td>
      <td></td>
    </tr>
    <tr>
      <td>sequence_variant</td>
      <td>bits</td>
      <td></td>
      <td>float</td>
      <td>float(12,11)</td>
      <td>785801</td>
      <td></td>
    </tr>
    <tr>
      <td>t2tc</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>t2tc</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>t2tc</td>
      <td>nucleic_acid_id</td>
      <td></td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>id</td>
      <td>The target identifier used within the Target Central Resource Database. The id is unique in the target table.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20412</td>
      <td>Generated for each new version of TCRD.</td>
    </tr>
    <tr>
      <td>target</td>
      <td>name</td>
      <td></td>
      <td>Calmodulin-binding transcription activator 2</td>
      <td>varchar(255)</td>
      <td>20405</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>ttype</td>
      <td></td>
      <td>Single Protein</td>
      <td>varchar(255)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>description</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>comment</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>tdl</td>
      <td>Target development level, as defined at http://juniper.health.unm.edu/tcrd/</td>
      <td>Tbio, Tclin, Tdark, Tchem</td>
      <td>enum('Tclin','Tchem','Tbio','Tdark')</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>idg</td>
      <td></td>
      <td>boolean</td>
      <td>tinyint(1)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>fam</td>
      <td>Major target family</td>
      <td>Enzyme, GPCR, IC, Kinase</td>
      <td>enum('Enzyme','Epigenetic','GPCR','IC','Kinase','NR','oGPCR','TF','TF; Epigenetic','Transporter')</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td>target</td>
      <td>famext</td>
      <td></td>
      <td>TF</td>
      <td>varchar(255)</td>
      <td>139</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>241680</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>itype</td>
      <td>foreign key to column name on table info_type.</td>
      <td>UniProt Function</td>
      <td>varchar(255)</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>target_id</td>
      <td>foreign key to column id on table target.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>1791</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>nucleic_acid_id</td>
      <td></td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>string_value</td>
      <td></td>
      <td>Catalyzes the last step in the trans-sulfuration pathway from methionine to cysteine. Has broad substrate specificity. Converts cystathionine to cysteine, ammonia and 2-oxobutanoate. Converts two cysteine molecules to lanthionine and hydrogen sulfide. Can also accept homocysteine as substrate. Specificity depends on the levels of the endogenous substrates. Generates the endogenous signaling molecule hydrogen sulfide (H2S), and so contributes to the regulation of blood pressure. Acts as a cysteine-protein sulfhydrase by mediating sulfhydration of target proteins: sulfhydration consists of converting -SH groups into -SSH on specific cysteine residues of target proteins such as GAPDH, PTPN1 and NF-kappa-B subunit RELA, thereby regulating their function.</td>
      <td>text</td>
      <td>58520</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>integer_value</td>
      <td></td>
      <td>float</td>
      <td>decimal(12,6)</td>
      <td>32867</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>integer_value</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2609</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>date_value</td>
      <td></td>
      <td>date</td>
      <td>date</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>boolean_value</td>
      <td></td>
      <td>boolean</td>
      <td>tinyint(1)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_info</td>
      <td>curration_level</td>
      <td></td>
      <td>(cull)</td>
      <td>varchar(50)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>id</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>target_id</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>old_tdl</td>
      <td></td>
      <td></td>
      <td>varchar(10)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>new_tdl</td>
      <td></td>
      <td></td>
      <td>varchar(10)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>person</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>datetime</td>
      <td></td>
      <td></td>
      <td>timestamp</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>explanation</td>
      <td></td>
      <td></td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>application</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tdl_update_log</td>
      <td>app_version</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_contact</td>
      <td>id</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_contact</td>
      <td>contact_name</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_contact</td>
      <td>contact_email</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_contact</td>
      <td>date</td>
      <td></td>
      <td></td>
      <td>date</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_contact</td>
      <td>grant_integer</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_contact</td>
      <td>pi</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>id</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>contact_id</td>
      <td>foreign key to column id on table techdev_contact.</td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>comment</td>
      <td></td>
      <td></td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>publication_pcmid</td>
      <td></td>
      <td></td>
      <td>varchar(255)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>publication_pmid</td>
      <td></td>
      <td></td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>resource_url</td>
      <td></td>
      <td></td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>techdev_info</td>
      <td>data_url</td>
      <td></td>
      <td></td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>id</td>
      <td>TCRD TIGA association ID</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>102407</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>11765</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>ensg</td>
      <td>Ensembl gene ID</td>
      <td>ENSG00000000460</td>
      <td>varchar(15)</td>
      <td>11761</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>efoid</td>
      <td>EFO ID (trait ID)</td>
      <td>EFO_0006937</td>
      <td>varchar(15)</td>
      <td>1441</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>trait</td>
      <td>Trait name (from EFO)</td>
      <td>optic disc area measurement</td>
      <td>varchar(255)</td>
      <td>1441</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>n_study</td>
      <td>Number of studies for this gene-trait association</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>38</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>n_snp</td>
      <td>Number of SNPs for this gene-trait association</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>56</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>n_snpw</td>
      <td>Number of SNPs for this gene-trait pair, weighted by SNP-gene distance</td>
      <td>float</td>
      <td>decimal(5,3)</td>
      <td>3170</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>geneNtrait</td>
      <td>Number of traits associated with this gene</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>91</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>geneNstudy</td>
      <td>Number of studies associated with this gene</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>139</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>traitNgene</td>
      <td>Number of genes associated with this trait</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>269</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>traitNstudy</td>
      <td>Number of studies associated with this trait</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>64</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>pvalue_mlog_median</td>
      <td>median(-log(pvalue))</td>
      <td>float</td>
      <td>decimal(6,3)</td>
      <td>2129</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>or_median</td>
      <td>median(OR)</td>
      <td>float</td>
      <td>decimal(8,3)</td>
      <td>2679</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>n_beta</td>
      <td>Number of beta values for this gene-trait association</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>92</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>study_N_mean</td>
      <td>mean(N) for studies supporting this association</td>
      <td>integer</td>
      <td>int(2)</td>
      <td>7938</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>rcras</td>
      <td>Relative Citation Ratio Aggregate Score</td>
      <td>float</td>
      <td>decimal(5,3)</td>
      <td>2429</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>meanRank</td>
      <td>mean rank for selected gene-trait association variables (see TIGA documentation for list)</td>
      <td>float</td>
      <td>decimal(18,12)</td>
      <td>21932</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga</td>
      <td>meanRankScore</td>
      <td>100 - percentile(meanRank)</td>
      <td>float</td>
      <td>decimal(18,14)</td>
      <td>21932</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga_provenance</td>
      <td>id</td>
      <td>TCRD TIGA provenance ID</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>167811</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga_provenance</td>
      <td>ensg</td>
      <td>Ensembl gene ID</td>
      <td>ENSG00000000971</td>
      <td>varchar(15)</td>
      <td>12158</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga_provenance</td>
      <td>efoid</td>
      <td>EFO ID (trait ID)</td>
      <td>EFO_0001365</td>
      <td>varchar(15)</td>
      <td>1452</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga_provenance</td>
      <td>study_acc</td>
      <td>GWAS Catalog study accession</td>
      <td>GCST001100</td>
      <td>varchar(20)</td>
      <td>3930</td>
      <td></td>
    </tr>
    <tr>
      <td>tiga_provenance</td>
      <td>pubmedid</td>
      <td>PubMed ID</td>
      <td>21665990</td>
      <td>int(11)</td>
      <td>2500</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_articlerank</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>53609232</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_articlerank</td>
      <td>importance_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>3015526</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_articlerank</td>
      <td>pmid</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2505076</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_articlerank</td>
      <td>rank</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>143637</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_disease</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>7604</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_disease</td>
      <td>doid</td>
      <td></td>
      <td>DOID:3121</td>
      <td>varchar(20)</td>
      <td>7604</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_disease</td>
      <td>name</td>
      <td></td>
      <td>gallbladder cancer</td>
      <td>text</td>
      <td>7604</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_disease</td>
      <td>summary</td>
      <td></td>
      <td>A biliary tract cancer that is located_in the gallbladder.</td>
      <td>text</td>
      <td>5065</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_disease</td>
      <td>score</td>
      <td></td>
      <td>float</td>
      <td>decimal(34,16)</td>
      <td>6848</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_importance</td>
      <td>doid</td>
      <td>foreign key to column doid on table tinx_disease.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>3015526</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_importance</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>17425</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_importance</td>
      <td>disease_id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>6778</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_importance</td>
      <td>score</td>
      <td></td>
      <td>float</td>
      <td>decimal(34,16)</td>
      <td>660313</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_novelty</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18032</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_novelty</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>18032</td>
      <td></td>
    </tr>
    <tr>
      <td>tinx_novelty</td>
      <td>score</td>
      <td></td>
      <td>float</td>
      <td>decimal(34,16)</td>
      <td>15420</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon</td>
      <td>uid</td>
      <td></td>
      <td>CHEBI:138675</td>
      <td>varchar(20)</td>
      <td>18117</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon</td>
      <td>name</td>
      <td></td>
      <td>gas molecular entity</td>
      <td>text</td>
      <td>18113</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon</td>
      <td>def</td>
      <td></td>
      <td>Any of the passages admitting nutrient vessels to the medullary cavity of bone</td>
      <td>text</td>
      <td>12779</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon</td>
      <td>comment</td>
      <td></td>
      <td>A prepollex does not contain phalanges nor does it articulate with a metacarpal. It articulates with elements of the carpal (wrist) skeleton[AAO:DB]. editor note: TODO add articulation relations</td>
      <td>text</td>
      <td>1068</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon_parent</td>
      <td>uid</td>
      <td>foreign key to column uid on table uberon.</td>
      <td>UBERON:0003531</td>
      <td>varchar(20)</td>
      <td>18107</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon_parent</td>
      <td>parent_id</td>
      <td></td>
      <td>UBERON:0001419</td>
      <td>varchar(20)</td>
      <td>5747</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon_xref</td>
      <td>uid</td>
      <td>foreign key to column uid on table uberon.</td>
      <td>CL:0000000</td>
      <td>varchar(20)</td>
      <td>10934</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon_xref</td>
      <td>db</td>
      <td></td>
      <td>GO</td>
      <td>varchar(24)</td>
      <td>69</td>
      <td></td>
    </tr>
    <tr>
      <td>uberon_xref</td>
      <td>value</td>
      <td></td>
      <td>integer</td>
      <td>varchar(255)</td>
      <td>24122</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>282528</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>viral_protein_id</td>
      <td>foreign key to column id on table viral_protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>7463</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>5719</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>dataSource</td>
      <td></td>
      <td>P-HIPSTer</td>
      <td>varchar(20)</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>finalLR</td>
      <td></td>
      <td>float</td>
      <td>decimal(20,12)</td>
      <td>2170</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>pdbIDs</td>
      <td></td>
      <td></td>
      <td>varchar(128)</td>
      <td>44</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_ppi</td>
      <td>highConfidence</td>
      <td></td>
      <td>0,1</td>
      <td>tinyint(4)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_protein</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>12499</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_protein</td>
      <td>name</td>
      <td></td>
      <td>full_polyprotein 1..2119</td>
      <td>varchar(128)</td>
      <td>5020</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_protein</td>
      <td>ncbi</td>
      <td></td>
      <td>YP_009112712.1</td>
      <td>varchar(128)</td>
      <td>9709</td>
      <td></td>
    </tr>
    <tr>
      <td>viral_protein</td>
      <td>virus_id</td>
      <td>foreign key to column virusTaxid on table virus.</td>
      <td>1582094</td>
      <td>varchar(16)</td>
      <td>990</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>virusTaxid</td>
      <td></td>
      <td>integer</td>
      <td>varchar(16)</td>
      <td>1001</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>nucleic1</td>
      <td></td>
      <td>DNA,RNA, Retro</td>
      <td>varchar(128)</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>nucleic2</td>
      <td></td>
      <td>(+)ssRNA dsDNA retro-DNA (-)ssRNA ssDNA dsRNA retro-RNA</td>
      <td>varchar(128)</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>order</td>
      <td></td>
      <td>Mononegavirales</td>
      <td>varchar(128)</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>family</td>
      <td></td>
      <td>Rhabdoviridae</td>
      <td>varchar(128)</td>
      <td>28</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>subfamily</td>
      <td></td>
      <td></td>
      <td>varchar(128)</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>genus</td>
      <td></td>
      <td>Lyssavirus</td>
      <td>varchar(128)</td>
      <td>78</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>species</td>
      <td></td>
      <td>Gammapapillomavirus 7</td>
      <td>varchar(128)</td>
      <td>212</td>
      <td></td>
    </tr>
    <tr>
      <td>virus</td>
      <td>name</td>
      <td></td>
      <td>Human papillomavirus type 109</td>
      <td>varchar(128)</td>
      <td>1001</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>id</td>
      <td></td>
      <td>integer</td>
      <td>int(11)</td>
      <td>2600228</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>xtype</td>
      <td>foreign key to column name on table xref_type</td>
      <td>UniProt Keyword</td>
      <td>varchar(255)</td>
      <td>21</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>target_id</td>
      <td>foreign key to column id on table target</td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>protein_id</td>
      <td>foreign key to column id on table protein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>20412</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>nucleic_acid_id</td>
      <td></td>
      <td>(null)</td>
      <td>int(11)</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>value</td>
      <td></td>
      <td>KW-0677</td>
      <td>varchar(255)</td>
      <td>1458974</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>xtra</td>
      <td></td>
      <td>Repeat</td>
      <td>varchar(255)</td>
      <td>25703</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>dataset_id</td>
      <td>oreign key to column id on table dataset.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td>xref</td>
      <td>nhprotein_id</td>
      <td>foreign key to column id on table nhprotein.</td>
      <td>integer</td>
      <td>int(11)</td>
      <td>86258</td>
      <td></td>
    </tr>
    <tr>
      <td>xref_type</td>
      <td>name</td>
      <td></td>
      <td>L1000 ID</td>
      <td>varchar(255)</td>
      <td>21</td>
      <td></td>
    </tr>
    <tr>
      <td>xref_type</td>
      <td>description</td>
      <td></td>
      <td>CMap landmark gene ID. See http://support.lincscloud.org/hc/en-us/articles/202092616-The-Landmark-Genes</td>
      <td>text</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td>xref_type</td>
      <td>url</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td>xref_type</td>
      <td>eg_q_url</td>
      <td></td>
      <td>(null)</td>
      <td>text</td>
      <td>0</td>
      <td></td>
    </tr>
  </tbody>
</table>
  </body>
</html>
